keyboards/edgepromx/readme.md

```diff
@@ -0,0 +1,3 @@
+# Firmware for the EdgeProMX keyboard
```

Comment:
This needs to at least follow the [format](https://github.com/qmk/qmk_firmware/blob/master/data/templates/keyboard/readme.md) and given that it is a non-standard keyboard some degree of explainer as to its function and how to flash it would be useful.